### ISSUE TITLE
MaterialPBR::Texture::scale is an int?

### ISSUE THREAD

**Issue by Ziriax** [KhronosGroup/COLLADA2GLTF#138]
In the file `GLTFMaterial.h`, you have the following code:

```
        class MaterialPBR : public GLTF::Material {
	  public: 
		class Texture : public GLTF::Object {
		  public:
			int scale = -1;
			GLTF::Texture* texture = NULL;
			int texCoord = -1;

			void writeJSON(void* writer, GLTF::Options* options);
		};
```
I would expect `scale` to be a float? What is the purpose of the integer scale?

Also, according to the [GLTF 2.0 spec, only a normal texture should have a scale?](https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#normaltextureinfo)

Thanks,
Peter

**Comment by lasalvavida** [comment]
> I would expect scale to be a float? What is the purpose of the integer scale?

You are correct, it should be a float. I was doing index properties for the texCoord, etc. and must have just done the same there without thinking about it.

**Comment by Ziriax** [comment]
I have a pull request ready then :-)

**Comment by lasalvavida** [comment]
Closed by #140.
